org.eclipse.ice.commands/src/main/java/org/eclipse/ice/commands/RemoteFileBrowser.java

```diff
@@ -61,25 +52,23 @@ public RemoteFileBrowser(Connection connection, final String topDirectory) {
 	 * @param channel
 	 * @param topDirectory
 	 */
-	protected void fillArrays(String topDirectory, ChannelSftp channel) {
-
+	protected void fillArrays(String topDirectory, SftpClient channel) {
 		try {
 			// Make sure the top directory ends with the appropriate separator
 			String separator = "/";
 			// If the remote file system returns a home directory with \, then it
 			// must be windows
-			if (channel.getHome().contains("\\"))
-				separator = "\\";
+			// TODO: I don't know how to do this using Mina. Is it really needed anyway?
```

Comment:
This is only needed for windows use, to check whether or not the remote file system is using / as a path separator or \\. We could alternatively just do an `lstat` on the home directory and then figure out whether or not the returned directory contains `/` or `\\`

Reply:
The sftp protocol does not really support this. Directory separators are specified as "/", so I don't think you can rely on this in general.

Reply:
Okay, I will try and test this on a windows machine then. There might be an alternative way to get around this.